apps/docs/src/app/core/component-docs/moment-datetime-adapter/examples/moment-adapter-options-example.component.html

```diff
@@ -0,0 +1 @@
+<fd-datetime-picker placeholder="M/D/YYYY, h:mm a"></fd-datetime-picker>
```

Comment:
when a datetime is entered according to the placeholder, the field becomes invalid: 
<img src="https://user-images.githubusercontent.com/53509521/126476160-075a5c89-be3f-4929-b1b9-b27d2afbb8ab.png" width="400">
on removing the `,` the field becomes valid. Why does this happen?

Reply:
@kavya-b this is just a wrong placeholder at input, done